build-virtualbox.ps1

```diff
@@ -29,7 +34,7 @@ if ($results.length -gt 0) {
 }
 
 echo "Running Pester Tests"
-Invoke-Pester -OutputFile PesterTestResults.xml -OutputFormat NUnitXml -EnableExit
+Invoke-Pester -OutputFile PesterTestResults.xml -OutputFormat NUnitXml # -EnableExit
```

Comment:
Why was `-EnableExit` commented out?

Reply:
So my local tests wouldn't kill the window during debugging. I'll reinstate that